<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WizardAutoStopPage</class>
 <widget class="QWidget" name="WizardAutoStopPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>470</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Open Sans</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>198</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QRadioButton" name="radioNoMax">
     <property name="text">
      <string>Stop calculation manually</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>198</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>198</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QRadioButton" name="radioMaxCalculations">
     <property name="text">
      <string>Stop calculation after</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="SNumberEdit" name="lineMaxCalculations">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="labelMaxCalculations">
     <property name="text">
      <string>iterations</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>198</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>198</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <widget class="QRadioButton" name="radioMaxCoincidences">
     <property name="text">
      <string>Stop calculation after</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="SNumberEdit" name="lineMaxCoincidences">
     <property name="minimumSize">
      <size>
       <width>170</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>170</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="labelMaxCoincidences">
     <property name="text">
      <string>coincidences</string>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>198</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SNumberEdit</class>
   <extends>QLineEdit</extends>
   <header>suil_number_edit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
